internal/pkg/errors/error_message.go

```diff
@@ -76,6 +76,9 @@ const (
 	InvalidOperationValueErrorMsg = "invalid operation value: %s"
 	ExactlyOneSetErrorMsg         = "exactly one of %v must be set"
 
+	// iam group-mapping commands
+	GroupMappingNoOpUpdateErrorMsg = "one of `--description`, `--name`, or `--filter` must be set"
```

Comment:
If this only gets used in one package, let's define it in that package, not globally.